src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/tests/__snapshots__/GrammarTests.test.ts.snap

```diff
@@ -981,6 +1088,36 @@ exports[`Grammar tests Implicit Expressions Awaited property 1`] = `
 "
 `;
 
+exports[`Grammar tests Implicit Expressions Close curly suffix 1`] = `
+"Line: @DateTime.Now}
```

Comment:
Yeah, I'm confused about why we're explictly adding this kind of thing into the grammar, because it looks wrong.